catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveCommand.java

```diff
@@ -118,7 +124,8 @@ private Object executeRemoveFromStore() throws Exception {
             return null;
         }
 
-        DeleteRequestImpl request = new DeleteRequestImpl(ids.toArray(new String[numberOfMetacardsToRemove]));
+        DeleteRequestImpl request =
+                new DeleteRequestImpl(ids.toArray(new String[numberOfMetacardsToRemove]));
         DeleteResponse response = catalogProvider.delete(request);
 
         if (response.getDeletedMetacards()
```

Comment:
❗️ We should change the success and error messages. Since the number of IDs could be huge if the filter is very broad, printing out all of them is probably not a good idea. We could only keep track of the number of metacards deleted instead and only report that number.